jbmc/src/java_bytecode/java_bytecode_convert_method.cpp

```diff
@@ -2197,6 +2197,8 @@ void java_bytecode_convert_methodt::convert_invoke(
   // - The translated method could be an inherited protected method, hence
   //   accessible from the original caller, but not from the generated test.
   //   Therefore we must assume that the method is not accessible.
+  // We set opaque methods as final by default to prevent from erroneously
```

Comment:
:pick: "to prevent from" -> "to avoid" / "to prevent jbmc from"

Reply:
changed